helm/vitess/templates/_helpers.tpl

```diff
@@ -105,21 +105,31 @@ export EXTRA_MY_CNF="$FLAVOR_MYCNF:/vtdataroot/tabletdata/report-host.cnf:/vt/co
 # backup flags - expects config.backup
 #############################
 {{- define "backup-flags" -}}
+# set tuple values to more recognizable variables
```

Comment:
The flag comment strikes again. This is getting injected into the command line, hiding everything after.

Reply:
I hate the flag comment!